workflow/cylc/beaufort-sea-hudson-bay/README.md

```diff
@@ -0,0 +1,19 @@
+## Cylc
+To run the `cylc` workflow with the test data, run:
+```bash
+cylc stop beaufort-sea-hudson-bay/*       # stops any currently running workflows
+cylc validate .                           # check the cylc configuration
+cylc install . -n beaufort-sea-hudson-bay # installs the current version of the workflow
+cylc play beaufort-sea-hudson-bay         # runs the workflow
+cylc tui beaufort-sea-hudson-bay          # opens the text user interface
```

Comment:
What do you think about adding, if possible, a utility that does all of this in one line?

Reply:
You could use `cylc vip <name>` for the middle three lines. (See https://cylc.github.io/cylc-doc/latest/html/user-guide/cheat-sheet.html for details.) 
I tend to just copy and paste the code into the command prompt, so whether it's three lines or five doesn't make much difference for me. For this example, I don't think it's worthwhile to shorten the code.

Reply:
I agree it's not worthwhile for one-off runs of the same sample workflow. I am looking at this from the perspective of a user running this very workflow for a different region later.  Don't examples illustrate general use? The user would need to make 4 updates (lines 1, 3, 4, and 5) to copied code if they needed to run this workflow for a different region. Just looking at it from a more abstract point of view.

Reply:
That's a fair criticism. How about this? It's a bit more explicit, less repeaty.